components/script/dom/htmlinputelement.rs

```diff
@@ -63,16 +63,118 @@ const PASSWORD_REPLACEMENT_CHAR: char = '●';
 #[derive(Clone, Copy, JSTraceable, PartialEq)]
 #[allow(dead_code)]
 #[derive(MallocSizeOf)]
-enum InputType {
-    InputSubmit,
-    InputReset,
-    InputButton,
-    InputText,
-    InputFile,
-    InputImage,
-    InputCheckbox,
-    InputRadio,
-    InputPassword
+pub enum InputType {
```

Comment:
I took the liberty of removing the `Input` prefix from the enum members, as this seemed redundant. Let me know if that's not OK.